src/components/FormInput/FormInput.spec.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import FormInput from './FormInput';
+
+describe('FormInput', () => {
+  it('renders a label and input with Name of input and a span with a conditional errormesasage', () => {
+    render(<FormInput label="Restaurantname" />);
```

Comment:
You need to add all props here. `value` is now undefined, that's why your test is not working.
All of your props need to be passe:
```js
    {
      id: 2,
      name: 'address',
      type: 'text',
      errorMessage: 'Address should include Street and Number',
      label: 'Address',
      required: true,
      pattern: /[^A-Za-z\- . [0-9]$]/g,
    }
```